CHANGELOG.md

```diff
@@ -35,10 +35,18 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 - Added support for FP opcodes 94-102 thus removing the need for `AVM_DISABLE_FP=On` with OTP-22+
 - Added support for stacktraces
 - Added support for `utf-8`, `utf-16`, and `utf-32` bit syntax modifiers (put and match)
-
+- Added support for the Erlang `gen_event` module
+- Added `start_link` support for the `network` module
+- Added support for `erlang:monotomic_time/1`
```

Comment:
Should probably also add stub in erlang.erl, as well as addition to programmer’s guide.